listenbrainz/webserver/views/recommendations_cf_recording_api.py

```diff
@@ -0,0 +1,135 @@
+import listenbrainz.db.user as db_user
+import listenbrainz.db.recommendations_cf_recording as db_recommendations_cf_recording
+
+from listenbrainz.webserver.errors import APIBadRequest, APINotFound
+
+from flask import Blueprint, jsonify, request
+from listenbrainz.webserver.decorators import crossdomain
+from listenbrainz.webserver.rate_limiter import ratelimit
+
+recommendations_cf_recording_api_bp = Blueprint('recommendations_cf_recording_v1', __name__)
+
+
+@recommendations_cf_recording_api_bp.route("/user/<user_name>/artist")
+@crossdomain()
+@ratelimit()
+def get_recommendations(user_name):
+    """ Get recommendations for user ``user_name``.
+
+        A sample response from the endpoint may look like::
+
+        {
+          "payload": {
+
+            "last_updated": "Sat, 06 Jun 2020 08:18:50 GMT",
+
+            "<artist_type>": {
+
+              "recording_mbids": [
+                "526bd613-fddd-4bd6-9137-ab709ac74cab",
+                "a6081bc1-2a76-4984-b21f-38bc3dcca3a5",
+                "a6ad0205-6e96-416d-a4e8-edd1773dac09",
+                "d8783d03-8a3b-4269-8261-00709d2cfee8"
+              ]
+            },
+
+            "user_name": "unclejohn69"
+          }
+        }
+
+        <artist_type>: 'top' or 'similar'
```

Comment:
the docs say that artist_type is `top` or `similar` but the actual code returns `top-artists` and `similar-artists`, we should be consistent here.